packages/cli/constructs.js

```diff
@@ -1 +1,3 @@
-module.exports = require('./dist').constructs
+import { constructs } from './dist'
```

Comment:
I think this has to stay the way it was, or?

Reply:
I think we can use the `import`, but I'm trying it locally with different type of projects.

Reply:
Because this is a module type

Reply:
I'll revert this change for now.